pkg/controller/plan/kubevirt.go

```diff
@@ -1835,6 +1838,161 @@ func (r *KubeVirt) EnsurePersistentVolume(vmRef ref.Ref, persistentVolumes []cor
 	return
 }
 
+func GetOvaPvNfs(client client.Client, planName string, providerName string) (pv *core.PersistentVolume, found bool, err error) {
+	pv = &core.PersistentVolume{}
+	err = client.Get(
+		context.TODO(),
+		types.NamespacedName{
+			Name: fmt.Sprintf("ova-nfs-pv-%s-%s", providerName, planName),
+		},
+		pv,
+	)
+
+	if err != nil {
+		if k8serr.IsNotFound(err) {
+			return nil, false, nil
+		}
+		err = liberr.Wrap(err)
+		return
+	}
+	return
+}
+
+func GetOvaPvcNfs(client client.Client, planName string, planNamespace string, providerName string) (pvc *core.PersistentVolumeClaim, found bool, err error) {
+	pvc = &core.PersistentVolumeClaim{}
+	err = client.Get(
+		context.TODO(),
+		types.NamespacedName{
+			Name:      fmt.Sprintf("ova-nfs-pvc-%s-%s", providerName, planName),
+			Namespace: planNamespace,
+		},
+		pvc,
+	)
+
+	if err != nil {
+		if k8serr.IsNotFound(err) {
+			return nil, false, nil
+		}
+		err = liberr.Wrap(err)
+		return
+	}
+	return
+}
+
+func (r *KubeVirt) CreatePvForNfs() (err error) {
+	sourceProvider := r.Source.Provider
+	splitted := strings.Split(sourceProvider.Spec.URL, ":")
+	if len(splitted) != 2 {
+		r.Log.Error(nil, "NFS server path doesn't contains: ", "url", sourceProvider.Spec.URL)
+		return fmt.Errorf("bad source provider %s URL %s", sourceProvider.Name, sourceProvider.Spec.URL)
+	}
+	nfsServer := splitted[0]
+	nfsPath := splitted[1]
+
+	pvName := fmt.Sprintf("ova-nfs-pv-%s-%s", r.Source.Provider.Name, r.Plan.Name)
+	_, found, err := GetOvaPvNfs(r.Destination.Client, r.Plan.Name, r.Plan.Provider.Source.Name)
+	if err != nil {
+		r.Log.Error(err, "Failed to get ova PV")
+		return
+	}
+	if found {
+		r.Log.Info("The PV for OVA NFS exists", "PV", pvName)
+		return
+	}
+
+	labels := map[string]string{"providerName": r.Plan.Provider.Source.Name, "app": "forklift", "migration": r.Migration.Name, "plan": r.Plan.Name}
```

Comment:
```suggestion
	labels := map[string]string{"provider": r.Plan.Provider.Source.Name, "app": "forklift", "migration": r.Migration.Name, "plan": r.Plan.Name}
```
(unless we rely on `providerName` elsewhere)